ui/StatusQ/StatusQSources.cmake

```diff
@@ -0,0 +1,27 @@
+set(STATUSQ_DIR ${CMAKE_CURRENT_LIST_DIR})
+
+set(STATUSQ_HEADERS
+    ${STATUSQ_DIR}/include/StatusQ/statuswindow.h
+    ${STATUSQ_DIR}/include/StatusQ/typesregistration.h
+    ${STATUSQ_DIR}/include/StatusQ/QClipboardProxy.h
+    ${STATUSQ_DIR}/include/StatusQ/statussyntaxhighlighter.h
+    ${STATUSQ_DIR}/include/StatusQ/rxvalidator.h
+)
+
+set(STATUSQ_SOURCES
+    ${STATUSQ_DIR}/src/statuswindow.cpp
+    ${STATUSQ_DIR}/src/typesregistration.cpp
+    ${STATUSQ_DIR}/src/QClipboardProxy.cpp
+    ${STATUSQ_DIR}/src/statussyntaxhighlighter.cpp
+    ${STATUSQ_DIR}/src/rxvalidator.cpp
+)
+
+if(APPLE)
+    list(APPEND STATUSQ_SOURCES
+        ${STATUSQ_DIR}/src/statuswindow_osx.mm
```

Comment:
don't we have to prepend here @alexjba ?

Reply:
But please not how it was done before and how it changed. Earlier all cpp/mm files were globbed and compiled, now `statuswindow_osx.mm` and `statuswindow_other.cpp` are mutually exclusive, only one source can be compiled so append/prepend doesn't matter.

Reply:
Cool, thx for the explanation, it wasn't obvious from the changes :)